src/main/groovy/io/saagie/plugin/dataops/utils/SaagieUtils.groovy

```diff
@@ -352,6 +355,74 @@ class SaagieUtils {
         return buildRequestFromQuery(listProjectTechnologies)
     }
 
+//    ================= App request =====================
+
+    Request getAppDetailRequest(String appId) {
+        logger.debug('Generating App detail with id', appId)
+
+        def jsonGenerator = new JsonGenerator.Options()
+            .build()
+
+        def gqVariables = jsonGenerator.toJson([id: appId])
+
+        def getAppDetailQuery = gq('''
+            query labWebApp($id: UUID!) { labWebApp(id: $id) {    id    name    description    creationDate    isDeletable   storageSizeInMB    instances(limit: 1, checkInPipelineInstance: false) {      id     status     statusDetails     startTime      endTime      version {        number              }          }    versions {     number      creator      creationDate      number      isCurrent      releaseNote      dockerInfo {        image        dockerCredentialsId             }      exposedPorts {        name        port        isAuthenticationRequired       isRewriteUrl        basePathVariableName              }      storagePaths         }    alerting {      emails      statusList      loginEmails {        login        email             }          }    technology {     id  label         }      }}
+        ''', gqVariables)
+        return buildRequestFromQuery(getAppDetailQuery)
+    }
+
+    Request getAppListByNameAndIdRequest(String projectId) {
```

Comment:
can you rename it to more significant name ? you don't use the name as a parameter, you only use the `projectId`